clients/privacy-center/pages/api/fides-js.ts

```diff
@@ -91,6 +91,11 @@ export default async function handler(
   // Check if a geolocation was provided via headers or query param
   const geolocation = await lookupGeolocation(req);
 
+  // Check if a property_id was provided in the query params
+  const propertyId = Array.isArray(req.query.property_id)
+    ? req.query.property_id[0]
+    : req.query.property_id;
```

Comment:
This smells a bit funny, let's follow the standard pattern of `param={value}` instead 👍 
```suggestion
  const { property_id: propertyId } = req.query
```

I'm not 100% sure why the `isArray` call is there?

Reply:
Also - please update the `fides-js` E2E tests for this 👍